database/src/pool/sqlite.rs

```diff
@@ -957,13 +957,14 @@ impl Connection for SqliteConnection {
         include: Option<&str>,
         exclude: Option<&str>,
         runs: Option<i32>,
+        profile: Option<&str>
     ) {
         self.raw_ref()
             .prepare_cached(
-                "insert into pull_request_build (pr, complete, requested, include, exclude, runs) VALUES (?, 0, strftime('%s','now'), ?, ?, ?)",
+                "insert into pull_request_build (pr, complete, requested, include, exclude, runs, profile) VALUES (?, 0, strftime('%s','now'), ?, ?, ?, ?)",
```

Comment:
Why did you change SQL queries? I don't think that we need to change anything SQL related in this PR. Also, if we did, we would have to write a migration.